cavern/src/main/java/org/opencadc/cavern/files/FileAction.java

```diff
@@ -126,27 +155,170 @@ protected UserPrincipalLookupService getUpLookupSvc() {
         return upLookupSvc;
     }
 
-    protected PosixIdentityManager getIdentityManager() {
-        return identityManager;
+    @Override
+    public void initAction() throws Exception {
```

Comment:
This method isn't doing anything except calling intiNodeURI, so I suggest moving the code from initNodeURI into here.  Also, the debug line doesn't output anything useful.

Reply:
done